online_store/gt_sports.html

```diff
@@ -0,0 +1,100 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2>
+<h2 class="column" id="h2title2">Online Store</h2>
+<br><br><br><br><br>
+<p></p>
```

Comment:
not used => remove it.
It's always a good idea to remove not used/not needed code. Because then it's less places to have a bug in your code

Reply:
Removed.